recipes-wpe/wpeframework/wpeframework_git.bb

```diff
@@ -35,11 +35,11 @@ PACKAGECONFIG_append = " ${@bb.utils.contains('DISTRO_FEATURES', 'thunder_releas
 
```

Comment:
Can we we can get rid of thunder_BUILDTYPE flags like thunder_debug? Thunder should now correctly propagate build type across thunder components, right?

Reply:
as we agreed, I think it's not needed anymore

Reply:
Not, still it is needed, since we have to config wifi module based on thunder config